go.mod

```diff
@@ -20,6 +20,8 @@ replace github.com/eycorsican/go-tun2socks => github.com/getlantern/go-tun2socks
 // on this unmerged PR against gomobile - https://github.com/golang/mobile/pull/58.
 replace golang.org/x/mobile => github.com/oxtoacart/mobile v0.0.0-20220116191336-0bdf708b6d0f
 
+replace github.com/tetratelabs/wazero => github.com/refraction-networking/wazero v1.7.1-w
```

Comment:
I'm guessing this was done automatically when you ran `go get` for `water`.  Is that right?  Just wanted to make sure this is intentional.

Reply:
It's not typical, but it could be due to some way water pulls in this `wazero` package (I haven't looked).

Reply:
No I had to do it manually, `wazero` has a different implementation from `tetratelabs/wazero` that contain different/custom functions. If we use `tetratelabs/wazero`, `water` will try to access functions that doesn't exist and panic (and their version has the functions they're expecting)

Reply:
I see!  Yeah, I see the same `replace` directive in the `water` library itself.  It's a shame that `go get` doesn't at least call this out when you pull in a new dependency (I tried it myself just to see).

I raised this issue with the WATER team [here on Slack](https://wdynhnkxvsdx.slack.com/archives/C06GJBFP8C8/p1719361086938969).  It's not a huge deal, but it seems like an easily resolvable pain point.  It doesn't look like `water` has any indirect dependencies on `wazero` so I believe they should be able to use their fork directly and do away with the `replace` statement.

Anyway, looks like we don't have a choice at the moment, so we can just leave it as is for now!  Thanks for explaining the situation.